gustaf/helpers/data.py

```diff
@@ -308,7 +308,7 @@ def __init__(self, helpee, **kwrags):
         Parameters
         -----------
         helpee: GustafBase
-        kwrags: **kwrags
+        kwrags: kwargs
```

Comment:
```suggestion
        **kwargs: kwargs
```

Reply:
Here you don't use the kwargs. Can it be removed, or do you need it?

Reply:
Nice catch, thanks,  it can be removed